bundles/org.openhab.binding.ojelectronics/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,158 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" bindingId="ojelectronics"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+	<!-- Bridge -->
+	<bridge-type id="ojcloud">
+		<label>OJ Electronics Bridge</label>
+		<description>Bridge to access OJ Electronic devices.</description>
+		<config-description>
+			<parameter name="userName" type="text" required="true">
+				<label>User Name</label>
+				<description>User Name for access to OJ Electronics cloud service.</description>
```

Comment:
I would really not repeat `OJ Electronics` so often, it feels like bad commercial like this :-)

Reply:
Yes, I actually use it relatively often.